Documentation/Books/Manual/Administration/Encryption/README.md

```diff
@@ -12,15 +12,21 @@ but when the data is already on disk, you also need protection.
 That is where the Encryption feature comes in. 
 
 The Encryption feature of ArangoDB will encrypt all data that ArangoDB 
-is storing in your database before it is written to disk.
+is storing in your database before it is written to disk. This feature applies to all ArangoDB Deployments.
 
 The data is encrypted with AES-256-CTR, which is a strong encryption
 algorithm, that is very suitable for multi-processor environments. This means that 
 your data is safe, but your database is still fast, even under load.
 
 Most modern CPU's have builtin support for hardware AES encryption, which makes it even faster.
 
-Note: The Encryption feature requires the RocksDB storage engine.
+Limitations
+===========
```

Comment:
This creates a level 1 headline. Should be level 2 I guess, so `---` but the other headlines seem to use `#` style headlines, so rather use `## Limitations` here for consistency.

Reply:
Yes you are correct, I will change it with `#`